docs/WindowsSupport.md

```diff
@@ -0,0 +1,83 @@
+# Windows support
+
+Using this extension on Windows requires additional setup steps. You can use one of two approaches:
+
+- [Run VS Code in Windows Subsystem for Linux](#wsl)
+- [Run VS Code in a Docker Container](#docker)
+
+## <a name="wsl"></a>Run VS Code in Windows Subsystem for Linux (WSL)
+
+### Prerequisites
+
+1. Install the [Windows Subsystem for Linux](https://docs.microsoft.com/windows/wsl/install-win10) along with your preferred Linux distribution.
+1. Install [Visual Studio Code](https://code.visualstudio.com/) on the Windows side (not in WSL).
+1. Install the [Remote Development extension pack](https://aka.ms/vscode-remote/download/extension).
```

Comment:
Incorrect list. Should be `1. 2. 3.`

Reply:
It is allowed in macrodown https://www.markdownguide.org/basic-syntax/#ordered-lists and I like it because we don't need to change all the list if we delete something something in the middle of the list for example.